### PR TITLE
📝 `Operator`: Draft Introduction and Outline for Operating a Convene Neighborhood

### PR DESCRIPTION
**Pull request by zspencer**
- https://github.com/zinc-collective/convene/issues/103

OK this is mostly stream of consciousness with some light tidying. Not all of this is probably necessary; but my inclination is to lean towards the underlying philosophy in `Operator` facing documentation, and take a more systems-oriented approach. This is two-fold:

1. Scaling `Operator`s is how we horizontally scale the amount of `People` `Convene` provides super-powers to
2. I don't want to shy away from the transformative nature of the project. Sometimes I feel like I'm a raving lunatic; but the reason I'm here is to eliminate oppressive systems by providing force-multipliers for laborers at solidarity organizations; and I would like to leverage the economic might of software to draw the people who are interested in that together.
3. I would hope that as we get more `Operator`s they will be able to draw in more `Contributor`s in connection with their `Neighbor`s, creating a virtuous cycle.

Very interested in feedback from folks, particularly around:

1. The perspective is intended to be a 10,000~1,000 ft view; starting at the high level then dropping down as it gets more nuanced, and back up. Does this seem high-level to the point that it loses all meaning?
2. Does this help you get an idea for who we are designing Convene for, from a socioeconomic niche perspective? Admittedly, it's a wide niche; with varied and distinct needs... Which is why I talk about as an "operating system" or "platform" rather than a  "framework" or even a "service"
3. The 1000 ft view is about when I would start referencing specific contracts, organizing models, regulation, web standards, or open protocols. Where would you like to see more things at the 1,000 ft view? What about less?

Drop it in a comment on the PR or send a DM!

### CODE REVIEW

File: OPERATING.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  - [As a Cooperative](#as-a-cooperative-1)
+  - [As a Person](#as-a-person-1)
+  - [As a Classic Corporation](#as-a-classic-corporation)
+- [Technology Details](#technology-details)
+  - [As a Cooperative](#as-a-cooperative-2)
+  - [As a Person](#as-a-person-2)
+  - [As a Classic Non-Profit, Government, or Corporation](#as-a-classic-non-profit-government-or-corporation)
+
+
+Convene is as much a socioeconomic project as it is a technological one. It serves as a platform for designing, building and hosting web software applications that grant super-powers to mutual-aid and mutual benefit solidarity economy organizations.
```
Comment (KellyAH) [inline OPERATING.md:17]: I agree this is a `stream of consciousness` and IMHO a bit to vague to easily understand.

Can the purpose/value of convene be explained in just 1 - 2 sentences in laymans/simple terms?

E.g. Convene provides a digital infrastructure space and tools that help small businesses/non-profits/grassroots orgs operate their businesses cheaper than enterprise vendors.

File: OPERATING.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Convene is deployed as a `Neighborhood` on commodity cloud infrastructure. It allows an `Operator`, (either a person or a worker and community-owned mutual-benefit technology cooperative) to support the digital operations of mutual-aid and mutual-benefit organizations (or `Neighbor`s).
+
+Where `Contributors` labor to research, design, test, and build software that delights `Neighbor`s and grants them superpowers, `Operators` labor to distribute this software to `People`. `Operators`:
```
Comment (KellyAH) [inline OPERATING.md:21]: An `Operator` honestly sounds more like the IT Admin of an org who signs up to manage the convene part of a business.

File: OPERATING.md (hunk 1 above)
Comment (KellyAH) [inline OPERATING.md:17]: `grant super-powers` is too vague.

**Review by KellyAH — CHANGES_REQUESTED** [review]
Sorry. My brain melted once I got to the `Social Responsibilities` section.

I can't tell if this is a "How customers can get started with convene" doc or a "operator duties & role" doc or something else.

There must be a way to explain what you are trying to communicate (the purpose/philosophy/goal/value of convene?) in a way that a 5-year-old or 96-year-old, non-techy person would understand.

**Comment by KellyAH** [comment]
Suggestions:
1️⃣ who is this doc's audience?
2️⃣ what problem is being solved by this doc?

File: OPERATING.md (hunk 2 above)
Comment (zspencer) [reply OPERATING.md:21]: Yep, that is exactly the service we are providing as the Operators of the Zinc neighborhood.

**Comment by zspencer** [comment]
The docs audience is the folks who have been reaching out to me about how they can launch their own digital services cooperative or leverage convene to provide services to their existing cooperatives clients. It's going to be long, it's going to be big; and it's not going to be useful for most people; but it is taking what I've been talking about with a half-dozen or so people  off and on over the last few years into something that exists.

I don't expect Contributors to have to worry about the `OPERATING` document, but I need it to live somewhere.

File: OPERATING.md (hunk 1 above)
Comment (zspencer) [reply OPERATING.md:17]: It is purposefully vague; since I can't tell folks what super-powers  their network of client cooperatives need. Again, Convene is an *Operating System*. It's entire value proposition is making it more cost effective for  Producers to provide the appropriate leverage to Consumers.

Comment (zspencer) [reply OPERATING.md:17]: That is what the README is for.

Comment (KellyAH) [reply OPERATING.md:17]: Oooo I like how you said that! ❤️ 

Maybe I can tweak it to this 👇  and update the readme!:

```
Convene is a cost-effective Operating System for building a Community Owned Digital Infrastructure. It provides small businesses/non-profits with the online tools and digital presence needed to run their operations and better serve their customers.

It is a more ethical and affordable alternative to investor-owned online tools like SquareSpace, Etsy, Patreon, Zoom, Yelp, and more.`
```

File: OPERATING.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- Welcome `People` to the `Neighborhood` and support them in becoming a good `Neighbor`.
+- Help `Neighbor`s get the most from their `Neighborhood` through carework, gluework and technical support.
+
+This guide covers the social, economic, and technological aspects of operating a Convene `Neighborhood`. Each section starts by describing the shared context, and then splitting between the three kinds of `Operator`s
+
+- Worker-owned and Directed Cooperative, Mutual-Benefit, or Mutual Aid Organizations
+- Individual People
+- Legacy Organizations, such as traditional Non-profits, Governments, or Corporations
+
+## Social Responsibilities
```
Comment (KellyAH) [inline OPERATING.md:33]: Are these guidelines for how to use convene in a Socially responsible manner?

We can't enforce/control how people run their orgs. But the least we can do is make it clear where we stand and provide education and guidance.

File: OPERATING.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    - Illegal Content - https://en.wikipedia.org/wiki/Digital_Services_Act
+    - Monopoly Busting - https://en.wikipedia.org/wiki/Digital_Markets_Act
+
+### As a Cooperative
+
+### As a Person
+
+### As a Legacy Organization
+
+## Economic Model
```
Comment (KellyAH) [inline OPERATING.md:65]: I assume this section would explain how to use convene in a cost-effective way by providing finance guidelines, legal FYI, etc.

File: OPERATING.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+### As a Cooperative
+
+### As a Person
+
+### As a Classic Corporation
+
+## Technology Details
```
Comment (KellyAH) [inline OPERATING.md:73]: Maybe the easiest part to write is what is needed 1st.

The instructions for how the current convene customers can launch their own digital services cooperative or leverage convene to provide services to their existing cooperatives clients.

And the doc iterates based on customer feedback.

**Review by anaulin — APPROVED** [review]
I really like the intro section as a way of laying out the high-level intention behind Convene, and also introducing different roles. Seems like that might be good in a general "overview of Convene" document as well.

I'm unclear about the purpose of the "Social Responsibilities" section. I do not want us to be prescriptive about what precise values we expect from Operators I would rather we stated what our recommendations are, and point folks at other places to learn more. If there are things that we consider to be non-negotiable, those should be clearly stated upfront, probably _before_ folks get to the "Operating" manual.

On a general style note, personally I would prefer to use simpler language as much as possible. For example, I don't think it is necessary to say "mutual-aid and mutual-benefit organizations"; we can have one intro sentence about how these are the types of organizations we want to support, and then refer to them as "organizations" henceforth. (Very personally, I see the tendency to use jargon and "special language" as a subtle power move that separates the speaker from anyone "not good enough" to understand the language used, so I try to stay away from that.)

All that said, I also think that this is ok to merge and then iterate (maybe with some TODOs or whatever). I also don't want us to get bogged down in writing The Perfect Document here; as @KellyAH said, this will get into better shape once we have real Operators that have real questions and help us iterate on the docs.

**Comment by zspencer** [comment]
Thanks both! I'm going to let this sit for a week and then do a revision that focuses on:

1. Plainer language
2. Organizing it by kind of operator (i.e. Person or Organization)

I think there does need to be something about the philosophy we would like operators to espouse; and that there is power in both the words we use to describe our intentions and offer guidance as well as who we name as the elders we look to for wisdom.

**Review by KellyAH — APPROVED** [review]
it's a good start!

File: OPERATING.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+Convene is as much a socioeconomic project as it is a technological one. It serves as a platform for designing, building and hosting web software applications that grant super-powers to mutual-aid and mutual benefit solidarity economy organizations.
+
+Convene is deployed as a `Neighborhood` on commodity cloud infrastructure. It allows an `Operator`, (either a person or a worker and community-owned mutual-benefit technology cooperative) to support the digital operations of mutual-aid and mutual-benefit organizations (or `Neighbor`s).
+
+Where `Contributors` labor to research, design, test, and build software that delights `Neighbor`s and grants them superpowers, `Operators` labor to distribute this software to `People`. `Operators`:
+
+- Nurture the `Neighborhood` by setting and maintaining healthy boundaries between `People`, `Neighbor`s and other `Neighborhood`s.
+- Welcome `People` to the `Neighborhood` and support them in becoming a good `Neighbor`.
+- Help `Neighbor`s get the most from their `Neighborhood` through carework, gluework and technical support.
```
Comment (daltonrpruitt) [inline OPERATING.md:25]: Looking up "gluework" (one word) seems to be only associated with organization names and a TikTok hastag. Should this be "glue work" (two words)? That appears to be the more accepted spelling, but that's based off only 1min of research.

File: OPERATING.md (hunk 1 above)
Comment (daltonrpruitt) [inline OPERATING.md:17]: I am a propoent of using the Oxford comma (would go after "building"). While it's not a hill I choose to die on, I want to mention it in case anyone else thinks it would help with reader comprehension. I don't want to comment everywhere it's not present in case avoiding it is an intentional decision. 
(I do want to note that "social, economic, and technological" does use it, but I don't know if consistency really matters in this particular case. 🤷)

File: OPERATING.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+This guide covers the social, economic, and technological aspects of operating a Convene `Neighborhood`. Each section starts by describing the shared context, and then splitting between the three kinds of `Operator`s
```
Comment (daltonrpruitt) [inline OPERATING.md:27]: I don't think this second phrase of the second sentence is independent, so I don't think it needs a comma before "and". However, it could be made independent with a bit of rephrasing.

Comment (daltonrpruitt) [inline OPERATING.md:27]: Should there be a ":" at the end of this sentence?

File: OPERATING.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+- Legacy Organizations, such as traditional Non-profits, Governments, or Corporations
+
+## Social Responsibilities
+
+- Praxis
+  - Anti-(Fascist, Colonial, Imperial, White Supremacist, Patriarchal, Ableist, Heteronormative)
+    - Self and Community Love (see hooks, Lorde, Morrison, amb)
+    - Prison and Police Abolition (See Davis, Lorde, hooks)
+    - Self Determination (See Friere, hooks, Angelou, Davis)
+    - Intersectionality (See hooks)
```
Comment (daltonrpruitt) [inline OPERATING.md:40]: Could you provide links or footnotes with links for these?

File: OPERATING.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+- Individual People
+- Legacy Organizations, such as traditional Non-profits, Governments, or Corporations
+
+## Social Responsibilities
+
+- Praxis
+  - Anti-(Fascist, Colonial, Imperial, White Supremacist, Patriarchal, Ableist, Heteronormative)
+    - Self and Community Love (see hooks, Lorde, Morrison, amb)
+    - Prison and Police Abolition (See Davis, Lorde, hooks)
+    - Self Determination (See Friere, hooks, Angelou, Davis)
```
Comment (daltonrpruitt) [inline OPERATING.md:39]: "Self-Determination" (hyphen) This is based on quick search, so there may be reason to not hyphenate.

**Review by daltonrpruitt — APPROVED** [review]
I now realize that this is meant as a draft anyways, so will likely change later. I think our comments will persist after merging anyway, so nothing is lost by going ahead and approving the draft.

**Comment by zspencer** [comment]
Ugh I haven't looked at this so I put it to bed.
